src/ui/components/championSelect/championSelectComponent.ts

```diff
@@ -0,0 +1,79 @@
+///<reference path="../../_app.ts" />
+'use strict';
+
+import Component from "vue-class-component";
+const Vue = <vuejs.VueStatic>require("vue");
+
+import "../../css/champion-select.less";
+
+import CDNService from "../../services/cdnService.ts";
+import StaticService from "../../services/staticService.ts";
+import NetworkService from "../../services/networkService.ts";
+import SettingService from "../../services/settingService.ts";
+import ModalComponent from "../modal/modalComponent.ts";
+
+@Component({
+    template: require("./championSelectView.html")
+})
+export default class ChampionSelectComponent extends Vue {
+    private canPick: boolean;
+    private mySelectedChampion: number;
+    private remainingTime: number;
+    private players: Object[];
+    private leftTeam: number[];
+    private rightTeam: number[];
+    private timer: NodeJS.Timer;
+
+    ready() {
+        if (!NetworkService.currentConnection) {
+            this.$router.go("/login");
+        }
+        this.remainingTime = 90;
+        this.canPick = true;
+        this.timer = setInterval(this.timerStep, 1000);
```

Comment:
`timerStep` is not bound to this. Use `() => this.timerStep()` or `this.timerStep.bind(this)` instead.